src/main/java/com/gregtechceu/gtceu/common/block/explosive/ITNTBlock.java

```diff
@@ -0,0 +1,35 @@
+package com.gregtechceu.gtceu.common.block.explosive;
+
+import com.gregtechceu.gtceu.common.entity.GTExplosiveEntity;
+import com.gregtechceu.gtceu.common.entity.ITNTEntity;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.state.BlockBehaviour;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+public class ITNTBlock extends GTExplosiveBlock {
```

Comment:
It's technically not required, but I'd strongly recommend renaming all the `ITNT...` classes to `IndustrialTNT...` - just to make it easily distinguishable from some interface for TNT, since we're using the `I...` naming convention for interfaces.